guacamole-ext/src/main/resources/org/apache/guacamole/protocols/vnc.json

```diff
@@ -58,6 +58,14 @@
                     "type"    : "BOOLEAN",
                     "options" : [ "true" ]
                 },
+                {
+                    "name" : "compress-level",
+                    "type" : "NUMERIC"
```

Comment:
The only valid options here are integers from 1-9, yeah? I wonder if these should be dropdowns instead of freeform fields.

Reply:
Yeah, I went back and forth about that, and then thought maybe it's a time to think about creating an option for the `NUMERIC` field that would allow placing limits on the ranges of the values.

Reply:
That's a neat idea, though I think it might make more sense when there's more options than this. This is quite a bit fewer options than e.g. font-size

```
{
    "name"  : "font-size",
    "type"  : "ENUM",
    "options" : [ "", "8", "9", "10", "11", "12", "14", "18", "24", "30", "36", "48", "60", "72", "96" ]
},```

Reply:
Yeah, I'm not sure this particular use-case warrants it, just something we might do for the future. Could limit things like port numbers, font sizes, timeout values, etc., to sane values without having to enumerate them the way the enum requires.

Reply:
Okay, converted to `ENUM` fields and added the English translations for the options.